src/mlpack/tests/ne_test.cpp

```diff
@@ -0,0 +1,691 @@
+/**
+ * @file ne_test.cpp
```

Comment:
You mean all NE tests? I'm not sure, if you implement another NE algorithm or like to test an NE methods on another task, the test file gets longer and longer. If you split the test file, and someone is only interested in the NEAT method, he could just take a look at the NEAT test file. let me know what you think. If you like the idea of a single test file, we can go with that solution.